README.md

```diff
@@ -170,20 +180,87 @@ platform, but it's also been tested on the OrangeCrab and ButterStick.
 
    For the Arty A7 A35, set `FLASH_ADDRESS` to `0x300000` and pass `-f a35`.
    ```
-   microwatt/openocd/flash-arty -f a100 build/microwatt_0/arty_a7-100-vivado/microwatt_0.bit
-   microwatt/openocd/flash-arty -f a100 dtbImage.microwatt.elf -t bin -a $FLASH_ADDRESS
+   sudo microwatt/openocd/flash-arty -f a100 build/microwatt_0/arty_a7-100-vivado/microwatt_0.bit
+   sudo microwatt/openocd/flash-arty -f a100 dtbImage.microwatt.elf -t bin -a $FLASH_ADDRESS
```

Comment:
Hmmm, instead of telling people to use sudo, I think it would be better to tell people about making sure their userid is a member of the dialout group.